mailet/base/src/main/java/org/apache/mailet/base/MailetUtil.java

```diff
@@ -109,4 +122,90 @@ public static boolean getInitParameter(MailetConfig config, String name, boolean
         }
         return result;
     }
+
+    public static boolean canSendAutomaticResponse(Mail mail) throws MessagingException {
```

Comment:
SIEVE vacation, JMAP vacation, SIEVE replies, AbstractNotify mailet should offer an option for that IMO... Large range of projects that all depends on /mailet. Seems logical then to locate it here.